cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -206,209 +226,222 @@ def create_settings(self):
+   image where the objects of interest have light and dark features.
+""" % globals())
 
         self.spline_threshold = cps.Float(
-                "Background threshold", 2, minval=.1, maxval=5.0, doc="""
-            <i>(Used only if %(SM_SPLINES)s are selected for the smoothing method and
-            spline parameters are not calculated automatically)</i><br>
-            This setting determines the cutoff used when excluding
-            foreground pixels from consideration. On each iteration,
-            the method computes the standard deviation of background
-            pixels from the computed background. The number entered in this
-            setting is the number of standard deviations a pixel can be
-            from the computed background on the last pass if it is to
-            be considered as background during the next pass.
-            <p>
-            You should enter a higher number to converge stabily and slowly
-            on a final background and a lower number to converge more
-            rapidly, but with lower stability. The default for this
-            parameter is two standard deviations; this will provide a fairly
-            stable background estimate.""" % globals())
+            "Background threshold", 2, minval=.1, maxval=5.0, doc="""\
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method
+and spline parameters are not calculated automatically)*
+This setting determines the cutoff used when excluding foreground
+pixels from consideration. On each iteration, the method computes the
+standard deviation of background pixels from the computed background.
+The number entered in this setting is the number of standard
+deviations a pixel can be from the computed background on the last
+pass if it is to be considered as background during the next pass.
+
+You should enter a higher number to converge stabily and slowly on a
+final background and a lower number to converge more rapidly, but with
+lower stability. The default for this parameter is two standard
+deviations; this will provide a fairly stable background estimate.
+""" % globals())
 
         self.spline_points = cps.Integer(
-                "Number of spline points", 5, 4, doc="""
-            <i>(Used only if %(SM_SPLINES)s are selected for the smoothing method and
-            spline parameters are not calculated automatically)</i><br>
-            This is the number of control points for the spline.
-            A value of 5 results in a 5x5 grid of splines across the image and
-            is the value suggested by the method's authors. A lower value
-            will give you a more stable background while a higher one will
-            fit variations in the background more closely and take more time
-            to compute.""" % globals())
+            "Number of spline points", 5, 4, doc="""\ 
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method and
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.